pages/bib/[id]/index.tsx

```diff
@@ -22,6 +21,7 @@ import {
   getBibQueryString,
   buildItemTableDisplayingString,
   isNyplBibID,
+  buildBibMetadataTitle,
```

Comment:
Totally irrelevant for this release, but is there any appetite to move these utils onto the bib model?